catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/RegistryConstants.java

```diff
@@ -16,6 +16,8 @@
 public class RegistryConstants {
     public static final String REGISTRY_TAG = "registry";
 
+    public static final String REGISTRY_TAG_INTERNAL = "registry.internal";
```

Comment:
For my own sanity if nothing else, we could still follow our naming conventions here. 😄 `namespace.hyphenated-descriptor`. Nesting namespaces is discouraged.

I'll check and see if we have our naming conventions documented anywhere.